service/src/main/java/org/databiosphere/workspacedataservice/dao/RecordDao.java

```
@@ -438,6 +438,9 @@ private Object getValueForSql(Object attVal, DataTypeMapping typeMapping) {
 		if(typeMapping.isArrayType()){
 			return getArrayValues(attVal, typeMapping);
 		}
+		if(attVal instanceof List<?> list && typeMapping == DataTypeMapping.STRING){
```

This change will cause the list data to be stringified and successfully stored.  Alternatively we could throw an error and inform the user that they're trying to add array data to plain string column.